### PR TITLE
fix #60774 canyoublockit.com

### LINKED ISSUE #60774: canyoublockit.com
### Issue URL (Ads)
 [https://canyoublockit.com/extreme-test/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcanyoublockit.com%2Fextreme-test%2F)
### Comment
Username: @dhandavivek
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | Default
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies<br/><br/><b>Security:</b><br/> Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion, <br/>AdGuard DNS, <br/>BarbBlock

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#60774

### CODE REVIEW

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -156,6 +156,7 @@
 !
 ||puddingreluctantly.com^
 ||rapiddictionarytemporarily.com^
+||xuossk3tu6hb.com^$third-party
```
Comment (Alex-302) [inline EnglishFilter/sections/adservers.txt:159]: yep, real ad server

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -243,6 +243,8 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 rawdevart.com##.text-center > .d-inline-block.my-3
 myfamporn.com##.wps-player__happy-inside-zone
 myfamporn.com##.happy-under-player
+canyoublockit.com##a[href="ad.com"]
+canyoublockit.com##iframety[src^="//ds88pc0kw6cvc.cloudfront.net/"]
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:247]: error in tag name

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:247]: yep, it should be `iframe`

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -259,6 +259,9 @@ jizzbunker.net##.panel-body > div[data-zone]
 pinflix.com,pornrox.com##.advertisement-link
 rawdevart.com##.text-center > .d-inline-block.my-3
 myfamporn.com##.wps-player__happy-inside-zone
+canyoublockit.com##a[href="ad.com"]
+canyoublockit.com##iframe[src^="//ds88pc0kw6cvc.cloudfront.net/"]
+||xuossk3tu6hb.com^$domain=canyoublockit.com
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:264]: Why? This is ad server.

**Pull request closed without merging**